.github/workflows/dbx.yml

```diff
@@ -24,19 +29,23 @@ jobs:
         path: secureboot_objects
 
     - name: Check if DBX is out-of-date
+      id: check_hash
```

Comment:
is this needed?

Reply:
Will be removed.

Reply:
Final commit: https://github.com/Dasharo/edk2/pull/220/commits/80120f4f0bdf3cf6498a72c68b1318c448ccaf79